bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,237 @@
+
+        elif user is None:
+            # just making sure that the random dosent pick up the same user(ctx.author)
+            USER_LOVEFEST.remove(ctx.author)
+            user = random.choice(USER_LOVEFEST)
+            USER_LOVEFEST.append(ctx.author)
+
+            if valentine_type is None:
+                valentine, title = self.random_valentine()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} {title} {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['p', 'poem']:
+                valentine = self.valentine_poem()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A poem dedicated to {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['c', 'compliment']:
+                valentine = self.valentine_compliment()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A compliment for {user.display_name}{EMOJI_1}",
+                    color=Colours.pink
+                )
+            else:
+                # in this case, the user decides to type his own valentine.
+                valentine = valentine_type
+                embed = discord.Embed(
+                    title=f'{EMOJI_1}A message for {user.display_name}{EMOJI_2}',
+                    color=Colours.pink
+                )
+
+            embed.description = f'{valentine} \n **{EMOJI_2}From anonymous{EMOJI_1}**'
+            await ctx.author.send(f"Your message has been sent to {user}")
+            await user.send(embed=embed)
+
+    @staticmethod
+    def random_valentine():
+        """
+            grabs a random poem or a compliment (any message)
+        """
+        with open(Path('bot', 'resources', 'valentines', 'bemyvalentine_valentines.json'), 'r', encoding="utf8") as f:
+            valentines = load(f)
+            valentine_poem = random.choice(valentines['valentine_poems'])
+            valentine_compliment = random.choice(valentines['valentine_compliments'])
+            random_valentine = random.choice([valentine_compliment, valentine_poem])
+            if random_valentine == valentine_poem:
+                message_type = 'A poem dedicated to'
+            else:
+                message_type = 'A compliment for '
+            return random_valentine['message'], message_type
+
+    @staticmethod
+    def valentine_poem():
+        """
+            grabs a random poem
+        """
+        with open(Path('bot', 'resources', 'valentines', 'bemyvalentine_valentines.json'), 'r', encoding="utf8") as f:
+            valentines = load(f)
+            valentine_poem = random.choice(valentines['valentine_poems'])
+            return valentine_poem['message']
+
+    @staticmethod
+    def valentine_compliment():
+        """
+            grabs a random compliment
```

Comment:
Indent needs to be corrected, capitalise first word, and a period probably should be added to the end.